ofnetSvcProxy.go

```diff
@@ -518,6 +545,11 @@ func (proxy *ServiceProxy) SwitchConnected(sw *ofctrl.OFSwitch) {
 	proxy.ofSwitch = sw
 
 	log.Infof("Switch connected(svcProxy).")
+
```

Comment:
Do we need to trigger polling here? The trigger from HandlePkt should be enough?